modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -209,6 +209,33 @@ mirror:
     targetCatalog: my-namespace/my-operator-catalog
 ----
 
+// New example; multi-arch; all versions
+[discrete]
+[id="oc-mirror-image-set-examples-multi-arch_{context}"]
+== Use case: Including all versions of OpenShift Container Platform from a minimum to the latest
```

Comment:
+1 to unique heading.

Reply:
I think the ask was to put this content under the existing heading and not to create a new heading. 
Since I am taking this over, I will create a new PR.